src/deephaven/plot/express/data/data_generators.py

```diff
@@ -1,18 +1,19 @@
 from __future__ import annotations
 
+import pandas as pd
 from deephaven.replay import TableReplayer
-from deephaven.time import parse_instant
 from deephaven.table import Table
 from deephaven import empty_table
 from deephaven.time import (
-    parse_instant,
-    plus_period,
-    MINUTE,
-    seconds_to_nanos,
+    to_j_instant,
+    to_pd_timestamp,
 )
 from deephaven.updateby import rolling_sum_tick, ema_tick
 import random, math
 
+SECOND = 1000000000  #: One second in nanoseconds.
```

Comment:
For long numbers like this, I recommend separating the groups of zeroes with `_` so it's easier to visually see how many zeroes you've got, e.g.
```python
SECOND = 1_000_000_000
```
https://peps.python.org/pep-0515/